metricbeat/module/uwsgi/status/_meta/data.json

```diff
@@ -0,0 +1,53 @@
+{
+    "@timestamp": "2011-11-11T00:00:00.000Z",
+    "metricset": {
+      "rtt": 3069,
+      "name": "stat",
+      "module": "uwsgi"
+    },
+    "uwsgi": {
+      "status": {
+        "total": {
+          "requests": 102,
+          "exceptions": 1,
+          "write_errors": 0,
+          "read_errors": 0,
+          "pid": 1
+        },
+
```

Comment:
Did you generate this file or add it manually? I kind of wonder how this newline here happened.

Reply:
I made this file manually, actually uwsgi.status contains only one of the nested fields (total, worker or core)

Reply:
Normally we generate these fields automatically with one of the tests like here for example: https://github.com/elastic/beats/blob/master/metricbeat/module/kibana/status/status_integration_test.go#L27 Like this we assure that the file stays up-to-date if we change mappings. Perhaps you could open a follow up PR with a file that is auto generated?